soroban-token-spec/src/lib.rs

```diff
@@ -106,3 +107,53 @@ impl Token {
         panic!("calling into interface");
     }
 }
+
+impl Token {
```

Comment:
I was incorrect, the `spec_xdr()` should be a crate function, not a function on `Token`, because the `Token::spec_xdr` if it ever exists should only contain the XDR for the contract functions, not the contract types. The namespace of the types and functions together is the crate, so we should define the function there.

```suggestion
```

Reply:
I don't get this, why is it wrong to include it in the contract?